apps/frontend/src/hooks/aave/useAaveRates.tsx

```diff
@@ -0,0 +1,148 @@
+import { useMemo, useState } from 'react';
+
+import { BigNumber, Contract } from 'ethers';
+import { formatUnits } from 'ethers/lib/utils';
+import { useSearchParams } from 'react-router-dom';
+
+import { useAccount } from '../useAccount';
+import rateStrategy from './ReserveStrategy-rateStrategyStableOne.json';
+import { BIG_NUMBER_PRECISION_TWENTY_SEVEN } from './constants';
+import { useAaveReservesData } from './useAaveReservesData';
+
+export interface IRatesDataResult {
+  currentUsageRatio: string;
+  baseStableBorrowRate: string;
+  optimalUsageRatio: string;
+  baseVariableBorrowRate: string;
+  variableRateSlope1: string;
+  variableRateSlope2: string;
+  stableRateSlope1: string;
+  stableRateSlope2: string;
+  //baseStableRateOffset: string;
+  stableRateExcessOffset: string;
+  optimalStableToTotalDebtRatio: string;
+  underlyingAsset: string;
+  name: string;
+  symbol: string;
+  decimals: string;
+}
+
+function calculateUtilizationRate(
+  totalDebt: string,
+  availableLiquidity: string,
+): string {
+  // Convert inputs to BigNumber
+  const _totalDebt: BigNumber = BigNumber.from(totalDebt);
+  const _liquidity: BigNumber = BigNumber.from(availableLiquidity);
+
+  return _totalDebt.div(_totalDebt.add(_liquidity)).toString();
+}
+
+export const useAaveInterestRatesData = (): {
+  data: IRatesDataResult | null;
+  error: string | null;
+  loading: boolean;
+} => {
+  const [data, setData] = useState<any>(null);
+  const [loading, setLoading] = useState<boolean>(true);
+  const [error, setError] = useState<string | null>(null);
```

Comment:
remove `any` also verbose `boolean`. And maybe handle error here with `useNotifyError` instead of returning it